### PR TITLE
Add scs-public environment

### PR DESCRIPTION
**Pull request by joshmue**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,58 +2,121 @@
...
-GITHUB_CLIENT_ID=<client_id>
-GITHUB_CLIENT_SECRET=<client_secret>
+```bash
+kubectl --context kind-status-page apply -k kubernetes/environments/kind/prerequisites
 ```
 
-As well as a client config in [`kubernetes/environments/kind/dex/config.yaml`](kubernetes/environments/kind/dex/config.yaml):
+#### Persistent Volume Claims
+
+`kind` does not provide a way to create a persistent volume for persistent volume claims. To be able to satisfy volume claims, rancher's `local-path-storage` is deployed locally.
```
Comment (joshmue) [inline README.md]: `kind` already deploys `local-path-storage` out of the box, doesn't it? So there should not be any need to deploy it as custom "prerequisite"?

File: README.md [hunk 2]
```diff
@@ -2,58 +2,121 @@
...
+  "access_token": "<your-access-token>",
+  "token_type": "bearer",
+  "expires_in": 86399,
+  "refresh_token": "<your-refresh-token>",
+  "id_token": "<your-id-token>"
+}
 ```
 
-### Persistent Volume Claims
+**NOTE**: Be **extreme careful** with your **code** and **client secret** as it represents **credentials**!
```
Comment (joshmue) [inline README.md]:
```suggestion
**NOTE**: Be **extremely careful** with your **code** and **client secret** as they represent **credentials**!
```

File: README.md [hunk 3]
```diff
@@ -2,58 +2,121 @@
...
+curl -sL \
+-X POST \
+-H 'Authorization: Bearer <your-id-token>' \
+-H 'Content-Type: application/json' \
+-d '{"displayName":"Test-Component"}'\
+https://status.k8s.scs.community/components
+```
 
-`kind` also does not provide a way to create a persistent volume for persistent volume claims. To be able to satisfy volume claims, rancher's `local-path-storage` is deployed locally, too.
+**NOTE**: Be **extreme careful** with your **id token** and **access token** as it represents **credentials**!
```
Comment (joshmue) [inline README.md]:
```suggestion
**NOTE**: Be **extremely careful** with your **id token** and **access token** as they represent **credentials**!
```

**Comment by joshmue** [comment]
Cannot approve this PR, as technically I opened it. @joshuai96 Feel free to address my comments and to approve/merge, afterwards.

File: README.md (hunk 1 above)
Comment (joshuai96) [reply README.md]: When i started the deployment project, requesting a PV(C) kept hanging in `pending` state. Solution was to deploy the local path storage. `kind` now seems to work with their own provider. Removed the prerequisite.

**Review by joshuai96 — APPROVED** [review]
LGTM

**Pull request merged**